File: auriculoterapia_api/Auriculoterapia.Domain/Cita.cs
using System;

namespace Auriculoterapia.Domain
{
    public class Cita
    {
        public int Id {get; set;}
<<<<<<< HEAD
=======
        public Paciente Paciente {get; set;}
>>>>>>> master
        public DateTime Fecha {get; set;}
        public DateTime HoraInicioAtencion {get; set;}
        public DateTime HoraFinAtencion {get; set;}
        public string Estado {get; set;}

        public int PacienteId {get; set;}
        public Paciente Paciente {get; set;}
        
    }
}

File: Auriculoterapia/app/src/main/java/com/example/android/auriculoterapia_app/models/Cita.kt
package com.example.android.auriculoterapia_app.models

import java.sql.Date
import java.sql.Timestamp


data class Cita(

    var id: Int,
    var fecha: String,
    var horaInicioAtencion: String,
    var horaFinAtencion: String,
    var estado: String,
    var tipoAtencionId: Int,
    var tipoAtencion: TipoAtencion,
    var pacienteId: Int,
    var paciente: Paciente

)